validation-test/compiler_crashers_2_fixed/0036-rdar23719809.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-swift-frontend %s -parse
```

Comment:
The RUN line here should be `// RUN: not %target-swift-frontend %s -parse` (note the `not`). This asserts that the parse will fail but not crash.